hack/lib/golang.sh

```diff
@@ -24,6 +24,7 @@ readonly KUBE_GOPATH="${KUBE_OUTPUT}/go"
 # If you update this list, please also update build/release-tars/BUILD.
```

Comment:
Done and done. I could create a new target group like `KUBE_EXTRA_TARGETS` just for this binary, but it seemed overkill for something that's meant to be temporary. Plus, the fact that this target sticks out like a sore thumb in `KUBE_ALL_TARGETS` is a good reminder that we should remove it.